iothub/device/devdoc/architecture.md

```diff
@@ -2,43 +2,32 @@
+This document describes the internal architecture for the Azure IoT Hub Device SDK. Applications use `IotHubDeviceClient` and `IotHubModuleClient` which represent the two types of device identities. During object creation, a transport pipeline, following the _chain of responsibility_ pattern, is also created. All but the last element of the pipeline will perform alterations of application requests. The last element, based on `TransportHandlerBase`, has the responsibility of converting IoT Hub request messages into the selected protocol and perform network I/O.
 
 ## Architecture diagram
 <!--arhitecture.puml-->
-![csharpDeviceClientSDKArchitecture](https://www.plantuml.com/plantuml/png/0/ZLLTRzem57r7uZzOVMAhaWUUfwg8q1PLQ0WwxTau5x1gxDI-OKFR_lliVBHnuZGy8C6zvvpxRUuDKetvIH1cZbHd2PmvWxdW876RFCyqEt84Xhc6yOW9QWmfdG-KeT8NCXxziPz_ur7jNae4BQaeOSu_7X5oIpAUWU0Ivie2VcDfN2OWu42HoyCJbQa0JRYKeAiVdd0pjMxObKfp0SsWXTDFDehkMlcDHwrye-Yh5oa8Q0Rh0kx8pOkcqcHA8rbNPT-uR3BANka1WlwwKzofxIc3K7CSe40H4iSV8bka275SEcx9zmpap5magmrpeGnubf-KuuFIxn858lKWI_F3q9s0EbpP5OBAoUKfcIJJ-PUFe5kGwU9i6z0GYUCIQZaLUpAV9WtI1XZmiv_QR6UBLQq2r2aJWE1fbrDmujMtoMgwQeWlf4k_oAaRnz81ZoerRNnbziAxsahx1coxfF6LNdvnwJV2MHu1EoBElgR8Zfwd3Dpr5HjC2DqPr7Q3INq7UveB_6GvAbu9tm3goxNWXTPANmjUQuMAAR5HkGCFqrg5l2zVOBlj4ilMSZvQAknp8Iu1IC8DK_RbO0Xf9z7PwIth3-P1Ls-8LqAV48TL0pVyxMyKzGxHSNe7N333ynpcXKSxp1kQyh12kqcEqMbSWGmDgXyTcbK_EZGkX-wv3_kAztUivqmivvfk3TwDrUbcMYpKlNr3x9_rVm00 "csharpDeviceClientSDKArchitecture")
+![csharpDeviceClientSDKArchitecture](https://www.plantuml.com/plantuml/png/bLJDKjim4BxxAJISQ9dumBd3G1CWEGGo9fcU5siJDCYIaDP0XlBkZHpNZiF6JO_Q_NxMgq--30IUoa8dkOOGnCJIk7py2G67Mg6Xv2CHuapCb4Ej30wj8Nod_NE5cOnGumf8cTKOZDJenSGOzFKX-KngZn6-ghpM5yc7Q3jJAqlTTc9ZVo0n67plAB28Leb7N8YJkGSJdFNjnvmaYOFy6LR8TaMr2sK8_H9ovBvtquyO8TAq5NWW84pJgducxXsQgx0sUNqUQXjmj_0B5DrOt_0hWDn5JC8YmAF6sccx344sQfLId8xEXzyQlmSqSeuQ2oI_RbNIj097Liq_7PwFMjgrYGuPvgXUSOeXmAepvKY5Fa2QJf6Uh0nxWdMLKM8wSVHjESFWh0aTW_cRCetwBL8yxx8NJW9XXd67A6VUElIai75eXuhC6L3-n8p5zD9qF9_FHDG3KjQ4tIbkbCQ5qnRa0uI5LvshjeNJIDXYEhTgNktpTWrDVPD_8Vu-cYMKclZCTQ53QmpcSRGgUbyeNnbP3CSnqjVlERged641plLfDW7c_UkLU0J2Etnhq9FbdQKCGQwFU0-r8Jm6yf3JldFj5n_-ZkUzsTEwtN1nMMgz_ehOn23VhXDbz_kkYXVNSU8tDdqXEW8s9kfLu_usFtuF1WTpaOYsnHBvb1mlm8Nc25fVXFgP_Odgf_AHOCiLWRssFW8NXtu2T-E9-EUrYC75mStpZKZfs_hXteRNktE-MLl-pyuZdDt7PtA9HfQ5_WC0 "csharpDeviceClientSDKArchitecture")
```

Comment:
Diagram doesn't include the instance of DefaultDelegatingHandler that goes first in the series.